src/components/Settings/SettingDialog.tsx

```diff
@@ -1,22 +1,18 @@
 import { FloatingPortal } from '@floating-ui/react';
-import {
-  ChangeEventHandler,
-  FC,
-  useCallback,
-  useEffect,
-  useRef,
-  useState,
-} from 'react';
+import { FC, useCallback, useEffect, useRef, useState } from 'react';
 
 import { useTranslation } from 'next-i18next';
 
+import { HighlightColor } from '@/src/types/common';
 import { Theme } from '@/src/types/settings';
 import { Translation } from '@/src/types/translation';
 
 import { useAppDispatch, useAppSelector } from '@/src/store/hooks';
 import { UIActions, UISelectors } from '@/src/store/ui/ui.reducers';
 
 import XMark from '../../../public/images/icons/xmark.svg';
+import ToggleFullWidth from '../Common/ToggleSwitch/ToggleSwitchLabeled';
```

Comment:
Still think that default is not good to use, because now this file not about toggleFullWidth(now it's generic) but it still named such

Reply:
I think both approaches have advantages and disadvantage.

Changed to named export.